### ISSUE TITLE
Cannot accurately click on objects when moving between displays with different resolutions/DPI scaling settings

### ISSUE THREAD

**Issue by mshiroma** [gorhill/uBlock#3422]
### Describe the issue
Moving a Chrome window between displays with different resolutions and DPI scaling settings does not update object locations and sizes with respect to mouse location. It appears that elements are laid out based on settings of the display set as the main display and not the display the Chrome window is on.

Scenario Information:
Laptop display: 2560x1440, scaling at 125%, main display
External display: 3840x2160, scaling at 150%

### Screenshot in which the issue can be seen
Attempting to click on elements on:
Laptop display:
![Laptop](https://user-images.githubusercontent.com/1925656/34729420-9d28dc86-f52a-11e7-84d9-b1d0f6c68bfe.jpeg)

External display:
![External](https://user-images.githubusercontent.com/1925656/34729422-9e3bd29a-f52a-11e7-8362-6edecb3c19c7.jpeg)

### Steps for anyone to reproduce the issue
Prerequisite: have two displays that have different resolutions and scaling settings

1. Move a Chrome window from one display to another
2. Open extension, attempt to click on an object.

### Your settings
- OS/version: Windows 10, 1607
- Browser/version: Chrome 59.0.3071.115 (Official Build) (64-bit)
- uBlock Origin version: 1.14.22

**Comment by gorhill** [comment]
Duplicate of #3116.

**Comment by gorhill** [comment]
The other issue, though the same, is for Firefox.

Here is the Chromium issue:
https://bugs.chromium.org/p/chromium/issues/detail?id=624992
